### PR TITLE
Format timestamp to RFC3339 in logs of go components

### PR DESCRIPTION
**Pull request by qibobo**
[#154198496]

### CODE REVIEW

**Comment by cdlliuy** [comment]
LGTM

File: src/autoscaler/logger/models.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	content, err := json.Marshal(tlf)
+	if err != nil {
+		_, ok1 := err.(*json.UnsupportedTypeError)
+		_, ok2 := err.(*json.MarshalerError)
+		if ok1 || ok2 {
+			tlf.Data = map[string]interface{}{"lager serialisation error": err.Error(), "data_dump": fmt.Sprintf("%#v", tlf.Data)}
+			content, err = json.Marshal(tlf)
+		}
+		if err != nil {
+			panic(err)
```
Comment (boyang9527) [inline src/autoscaler/logger/models.go]: do we really want to panic here?

File: src/autoscaler/logger/models_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+				Message:   "message",
+				LogLevel:  lager.INFO,
+				Data:      map[string]interface{}{"log-data": func() {}},
+			}
+		})
+
+		It("logs the serialization error", func() {
+			tlf := NewTimeLogFormat(log)
+			message := map[string]interface{}{}
+			fmt.Printf("===>%s\n", tlf.ToJSON())
```
Comment (boyang9527) [inline src/autoscaler/logger/models_test.go]: remove this line

File: src/autoscaler/logger/models_test.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+			logTime = time.Now()
+			log = lager.LogFormat{
+				Source:   "log-source",
+				Message:  "message",
+				LogLevel: lager.INFO,
+				Data:     map[string]interface{}{"log-data": "log-data"},
+			}
+		})
+		It("add RFC3339 format field log_time by the value 0 to output", func() {
+			tlf := NewTimeLogFormat(log)
```
Comment (boyang9527) [inline src/autoscaler/logger/models_test.go]: move this line to JustBeforeEach

File: src/autoscaler/logger/models.go (hunk 1 above)
Comment (qibobo) [reply src/autoscaler/logger/models.go]: The method "ToJSON" is copied from lager.LogFormat.

**Comment by boyang9527** [comment]
LGTM

**Pull request merged**